sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -266,6 +266,16 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
     }
   }
 
+  test("SPARK-32374: disallow setting properties for CREATE TEMPORARY VIEW") {
+    withTempView("myabcdview") {
+      val e = intercept[AnalysisException] {
```

Comment:
parser exception?

Reply:
interesting, the tests pass. Do you know why?

Reply:
Oh because `ParseException` extends `AnalysisException`. I will update this.